src/Symfony/Component/TypeInfo/Type/CollectionType.php

```diff
@@ -25,7 +25,7 @@
  *
  * @implements WrappingTypeInterface<T>
  */
-class CollectionType extends Type implements WrappingTypeInterface
+class CollectionType extends Type implements \Stringable, WrappingTypeInterface
```

Comment:
```suggestion
class CollectionType extends Type implements WrappingTypeInterface
```
`Type` is already `Stringable`.

Reply:
I blindly copied Fabbot suggestion, but you're absolutely right. Can I just ignore fabbot?

Reply:
In that case, yes I think so